### DISCUSSION TITLE
How do you merge Shawn.Utils.dll into 1Remote.exe?

### DISCUSSION THREAD

**Discussion by MWinhard** [1Remote/1Remote#695] [_Development]
It looks like you're merging Shawn.Utils.dll and other dlls into 1Remote.exe. I looked at your powershell skripts and the PostBuild Event in Ui.csproj, but I found no hint where/how you do this.

I need this for one of my own C# projects. I'd be happy if you point me in the right direction.

Thank you very much in advance.

Markus

**Comment by MWinhard** [comment]
I think I've found it. Is it the publish profile in Ui/Properties/PublishProfiles/x64-single.file.application.pubxml, that does the magic? Or do I need something else?

**Comment by VShawn** [comment]
all you need is publish the app with `produce single file` option.

![image](https://github.com/user-attachments/assets/3c6965aa-2449-4ce9-8e92-d67b8192deec)

**Comment by MWinhard** [comment]
Thank you very much
